libraries/support/azure/rest.rb

```diff
@@ -5,11 +5,9 @@
 
 module Azure
   class Rest
-    DEFAULT_HOST = 'https://management.azure.com'
-
     attr_reader :host, :resource, :credentials
 
-    def initialize(host: DEFAULT_HOST, credentials: {})
+    def initialize(host, credentials: {})
```

Comment:
Defaulting to management seems reasonable to me as most of our resources will speak to management by default.